Chart.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v2
+name: ack-chart
+description: A Helm chart containing all ACK charts as sub-charts
+type: application
+version: 1.0.0
```

Comment:
Since all of the controller charts are versioned with the leading `v` (e.g., `v1.2.3`), why not adopt the same here?

Reply:
Our inclusion of `v` is good practice for Go modules but actually not good practice for Helm charts - it breaks the semantic versioning systems for GitOps operators which expected only `X.Y.Z`. I'm torn between using the same broken logic as our previous controllers, for consistency, and making the move to the correct format.

Reply:
https://github.com/aws-controllers-k8s/community/issues/1715